ephypype/gather/gather_results.py

```diff
@@ -0,0 +1,88 @@
+"""Get pipeline results and figures."""
+
+# Authors: Annalisa Pascarella <a.pascarella@iac.cnr.it>
+#
+# License: BSD (3-clause)
+
+import glob
+import os.path as op
+
+
+def get_channel_files(workflow_path, workflow_name):
+    """Get channel files.
+
+    Parameters
+       workflow_path : str
+           Path of connectivity workflow
+       workflow_name : str
+           Name of the connectivity workflows
+
+    Returns
```

Comment:
also bad indentation and you need to add a line with dashes below this line. Otherwise, the docstring won't be formatted in bold